R/chromote_session.R

```diff
@@ -117,6 +134,24 @@ ChromoteSession <- R6Class(
         private$pixel_ratio <- value$result$value
       })
 
+      if (is.null(targetId)) {
+        # In Chrome v128-v133, `width` and `height` in `Target.createTarget` are
+        # ignored. For v134+ (latest stable as of 2025-03-06), they only have
+        # an effect when creating a new window, which is disabled by default.
+        # Regardless, width/height here override device emulation for this
+        # particular tab independently of the Chrome window dimensions. IOW,
+        # `window.innerWidth` and `.innerHeight` will match these dims.
```

Comment:
It'll be helpful to have some info on why `setDeviceMetricsOverride()` is better in general than allowing users to control creating new windows vs tabs, in case someone wonders why things are this way and is tempted to create new windows.

My understanding is that this way:

- has more consistent behavior across Chrome versions
- with this method, there's not actually any reason to create a new window
- it is possibly more memory efficient because Chrome never has to create a new window